features/base/test/test_umask.py

```diff
@@ -16,5 +16,15 @@
 
 # Run the test unit to perform the
 # final tests by the given artifact.
-def test_umask(client, file, dict):
+def test_umask(client, file, dict, non_gcp):
+    # Check /etc/login.defs
     file_content(client, file, dict)
+
+def test_umask_cmd(client, non_container):
+    # Additionally check umask via cmd
+    cmd = f"sudo su root -c umask"
```

Comment:
Out of curiosity, I stumbled over the `sudo su` idiom. I assume there is a reason to do it, but it's not obvious to me. Not important enough to add a comment I guess.

Reply:
The authentication is different
- `sudo su` authentication is handled via whatever sudo permissions the user has. In this case, the user is conceptional already privileged and does not need to enter a password for sudo
- `su` requires a password